pom.xml

```diff
@@ -698,6 +699,12 @@
         <artifactId>jackson-dataformat-yaml</artifactId>
         <version>${jackson.version}</version>
       </dependency>
+      <!-- JSON SChema -->
+      <dependency>
+        <groupId>com.networknt</groupId>
```

Comment:
VertX has a nice library to handle schema validation `io.vertx:vertx-json-schema`. Since we already have a bunch of vertx dependencies, maybe it would be beneficial to use it instead of pulling a dependency from a "different family". I don't feel strongly though. What do you think?

Reply:
Quickly looked at it and maybe the vertx library is too loaded for this. Not sure for eg. for what vertx-core is used.

Reply:
I was looking at that. We don't depend on vertx in the main code line, only for the server. Hence I thought this library, based on jackson, seemed like a better fit.